packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -34,7 +34,8 @@ class WidgetTester {
   void pump([ Duration duration ]) {
     if (duration != null)
       async.elapse(duration);
-    scheduler.beginFrame(new Duration(milliseconds: clock.now().millisecondsSinceEpoch));
+    scheduler.handleBeginFrame(new Duration(
+        milliseconds: clock.now().millisecondsSinceEpoch));
     async.flushMicrotasks();
   }
 
```

Comment:
While you're looking at this (but in a separate patch), if you can find a better way to handle the situation here where this test file calls  `ui.window.onBeginFrame = null;`, that would be great. When I added that code I couldn't really see a clean way to configure how the scheduler should be driven. This solution is bad because it is mucking around with a value that really the scheduler thinks it owns.